app/migrations/20240315000001-add-invitestatus-enum.cjs

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    // First create the enum type
+    await queryInterface.sequelize.query(`
+      DO $$ BEGIN
+        CREATE TYPE "enum_CityInvite_status" AS ENUM ('pending', 'accepted', 'canceled', 'expired');
+      EXCEPTION
+        WHEN duplicate_object THEN null;
+      END $$;
+    `);
```

Comment:
Again, do you need the email type here? As it is not referenced in the later query.